packages/pigment-react/src/processors/keyframes.ts

```diff
@@ -100,7 +102,9 @@ export class KeyframesProcessor extends BaseProcessor {
 
   generateArtifacts(styleObjOrTaggged: CSSInterpolation | string[], ...args: Primitive[]) {
     const { styles } = serializeStyles(
-      [styleObjOrTaggged as Interpolation<{}>, args],
+      args.length > 0
+        ? [styleObjOrTaggged as Interpolation<{}>, ...args]
+        : [styleObjOrTaggged as Interpolation<{}>],
```

Comment:
This condition is required to prevent `ILLEGAL_ESCAPE_SEQUENCE_ERROR` from emotion because it assume that `[]` is an expression.